### DISCUSSION TITLE
How do you edit the menu and welcome messages?

### DISCUSSION THREAD

**Discussion by Ebsbrown** [Steffo99/greed#186] [Q&A] [answered]
Hi how do you edit the menu and welcome message?

**Comment by Steffo99** [comment, marked as answer]
You can edit the menu buttons and welcome message by editing [the file in the `strings/` directory](https://github.com/Steffo99/greed/tree/master/strings) of the language you want to change.

For example, to change the welcome message for the `en` language, you'll have to edit the following lines:

https://github.com/Steffo99/greed/blob/8701071b30246c0046ffd4b42037c9c335dd0377/strings/en.py#L56-L60

Menu buttons are a bit below:

https://github.com/Steffo99/greed/blob/8701071b30246c0046ffd4b42037c9c335dd0377/strings/en.py#L132-L220
